### ISSUE TITLE
don't steal all the text file associations

### ISSUE THREAD

**Issue by CouldBeThis** [subethaedit/SubEthaEdit#203]
This program took the file association for every single text type document that exists and even those that do not exist. I installed it *months* ago and am still regularly finding new file types that open unexpectedly in it. 

Is there a way to un associate it?

**Comment by monkeydom** [comment]
Yeah, this is an unfortunate interface choice by the finder. All we do is offer that we understand these files, and then finder and the system decides how to distribute. You can change that for each file/type in the inspector in finder (cmd-i). 

And after you chose it, chose chose all to change it for the filetype in question:
![image](https://user-images.githubusercontent.com/5106/107618459-e4b2b580-6c51-11eb-8967-3f54e6ec2937.jpeg)

**Comment by CouldBeThis** [comment]
For anyone else who is looking, I found a solution to more easily change the file associations: https://github.com/Lord-Kamina/SwiftDefaultApps Changing them one by one as suggested is extremely tedious and cannot be done all at once because you have to wait for all the kinds of files to come to your attention.  

Seriously I have a bunch of text editors and none of the others ones installed before or since have had this behaviour so obviously there is something weird being done by this one. I lack the knowledge to say exactly what should be done differently but maybe youse can check out what all the others are doing differently.
